receiver/elasticapmreceiver/metadata.yaml

```
@@ -0,0 +1,8 @@
+type: elasticapm
```

```suggestion
type: elasticapm
scope_name: github.com/elastic/opentelemetry-collector-components/receiver/elasticapmreceiver
github_project: elastic/opentelemetry-collector-components
```

This will help mdatagen based auto generation code to use elastic's repository rather than the default upstream one.